static/src/app/library/library.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import leaflet from 'leaflet';
```

Comment:
Is this getting used anywhere? Is it possible to `lint` for unused imports like in python/scala?

Also, I'm not seeing a map on the `library` page